boxoffice/models/line_item.py

```diff
@@ -108,6 +108,23 @@ def current_assignee(self):
         return self.assignees.filter(Assignee.current == True).one_or_none()
 
 
+def get_availability(cls, item_ids):
+    """Returns a dict -> {'item_id': ('item title', 'quantity_total', 'line_item_count')}"""
+    items_dict = {}
+    item_tups = db.session.query(cls.id, cls.title, cls.quantity_total, func.count(cls.id)).join(LineItem).filter(
+        LineItem.item_id.in_(item_ids), LineItem.status == LINE_ITEM_STATUS.CONFIRMED).group_by(cls.id).all()
+    for item_tup in item_tups:
+        items_dict[unicode(item_tup[0])] = item_tup[1:]
```

Comment:
I'm afraid I can't do that. `item_tup` is a tuple with values, eg: `(UUID('17e37e78-178f-11e6-b989-21e4c39b7dd7'), u'T-shirt', 6, 5L)`.